docs(js): Add decompress flag to sourcemaps troubleshooting (#5287)
getsentry/sentry-docs: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx

The Sentry API currently only works with source maps and source files that are uploaded as plain text (UTF-8 encoded). If the files are uploaded in a compressed format (for example, gzip), they will be not be interpreted correctly.

> If you are using `sentry-cli` to upload your artifacts, starting version `2.4.0` you can add `--decompress` flag to your `sourcemaps upload` or `files upload` commands.

Reviewer: ```suggestion
If you are using `sentry-cli` to upload your artifacts, starting with version `2.4.0` you can add the `--decompress` flag to your `sourcemaps upload` or `files upload` commands.
```

> However, this sometimes also occurs with build scripts and plugins that produce pre-compressed minified files. For example, Webpack’s [compression plugin](https://github.com/webpack/compression-webpack-plugin). Otherwise, you’ll need to disable such plugins and perform the compression _after_ the generated source maps/source files have been uploaded to Sentry.

Reviewer: Adding "however" here makes it sound like 
```suggestion
Sometimes build scripts and plugins produce pre-compressed minified files (for example, Webpack’s [compression plugin](https://github.com/webpack/compression-webpack-plugin)). In these cases, you’ll need to disable such plugins and perform the compression **after** the generated source maps/source files have been uploaded to Sentry.
```
